daemon/daemon.go

```diff
@@ -1042,7 +1033,12 @@ func (daemon *Daemon) TagImage(newTag reference.Named, imageName string) error {
 	if err := daemon.referenceStore.AddTag(newTag, imageID, true); err != nil {
 		return err
 	}
-	daemon.EventsService.Log("tag", newTag.String(), "")
+
+	actor := eventtypes.Actor{
+		ID:   imageID.String(),
+		Name: newTag.String(),
+	}
+	daemon.EventsService.Log("tag", eventtypes.ImageEventType, actor)
```

Comment:
I believe all the `eventtypes.ImageEventType` events should include the image labels in `Actor.Attributes`. Previously this was being injected by `daemon.GetLabels()`